CONTRIBUTING.md

```diff
@@ -40,6 +40,33 @@ This document provides guidelines for contributing to the Ops Actions project.
 
     Use the Conventional Commits specification for semantic pull request titles.
 
+## ✅ Reviewing changes
+
+1. Ensure all changes follow our [best practices](./docs/best-practices.md).
+1. Check if there are any **breaking changes** that will require users to update their workflow calls:
+
+     - Add required input or secret.
+     - Update input type.
+     - Update output value.
+     - Rename input, secret or output.
+     - Remove input, secret or output.
+     - Rename workflow file.
+     - Remove workflow file.
+
+1. Ensure that the PR title follows the [Conventional Commits specificiation](https://www.conventionalcommits.org/en/v1.0.0/) and is using one of the following allowed types:
+
+      - **feat:** add or remove something (e.g. workflow, input, job, step or output)
+      - **fix:** fix something broken
+      - **refactor:** change something without adding, removing or fixing anything
+      - **docs:** document something
+      - **ci:** change CI/CD setup (e.g. non-reusable workflows or scripts)
+      - **chore:** everything else
+
+1. Audit third-party actions:
+
+      - Ensure commits are signed by trusted users (e.g. members of trusted organizations such as `actions`, `hashicorp`, `docker` or `Azure`).
+      - Perform a static security test (e.g. using [StepSecurity Action Advisor](https://app.stepsecurity.io/action-advisor)).
+
```

Comment:
Could a point describing how to verify if the changes (if relevant) have been tested or not? E.g. something like this maybe?

```suggestion
1. If the changes can be tested, verify that the PR author has documented this in the PR, or query if tests can be performed.
```

Reply:
Good point! I don't think it should be under "Audit third-party actions" though. I'll make a PR with a suggestion and re-request review.